core/src/main/java/org/lflang/target/property/PythonVersionProperty.java

```diff
@@ -0,0 +1,17 @@
+package org.lflang.target.property;
+
+/** Directive for specifying a specific version of the reactor runtime library. */
+public final class PythonVersionProperty extends StringProperty {
```

Comment:
We probably need some custom implementation of `validate` in this class, and check that no unsupported versions are selected.